tests/README-integration-tests.md

```diff
@@ -0,0 +1,15 @@
+# Integration Tests #
+
+The integration tests are written as Ansible playbooks targetted for all hosts.
+They follow the [Fedora's Standard Test
+Interface](https://fedoraproject.org/wiki/CI/Standard_Test_Interface) and can
+be run by the [Linux Sytem Roles Test
+Harness](https://github.com/linux-system-roles/test-harness). You can also run
+them directly, for example to run them against the [CentOS Cloud
+Images](https://cloud.centos.org/centos/7/images/) download one of them,
+install `standard-test-roles` from Fedora and execute `ansible-playbook`:
+
```

Comment:
Could you add more meat here? Just a short description of what are added files and subdirectories good for.

Reply:
done